### ISSUE TITLE
Beset: Run Master Control from its own project and not any longer from PVD

### ISSUE THREAD

**Issue by urapadmin** [arch-kiosk/arch-kiosk-office#2814]
Because there is only one MCP running for all projects on Beset it is now necessary to separate the MCP that the real projects use from the one that PVD is using. This is a preparation before we can start testing the new Kiosk version that supports time zones on Beset.

MCP, which is responsible for starting and managing jobs like synchronization, preparing for field, file export, etc., is currently running on the code basis of PVD, which was fine until now, because PVD usually has the most recent code. But now we need to be able to run different versions of MCP at the same time. So MCP gets its own project space on Beset and the production MCP must run from there.

**Comment by urapadmin** [comment]
@arch-kiosk/test I have done this on Beset. Please have a look at PVD and two production projects and make sure that job-related tasks still work properly. 
Job-related is everything you trigger for a dock in the hub but also synchronization or running housekeeping from the administration.

**Comment by luizaogs** [comment]
This doesn’t need a new version to be tested?

**Comment by urapadmin** [comment]
no, this was really just a configuration issue

**Comment by luizaogs** [comment]
Hmm I tried to do stuff on PVD, MKAP, and MADP at the same time and now they are all frozen. Which, maybe I shouldn't have done it all at the same time but I expect they shouldn't have frozen anyway? I thought it first it was bad hotel internet but I am able to access all other websites, just Kiosk is not loading for me.

**Comment by urapadmin** [comment]
FileMaker completely stuck. Happens, and usually under stress. I do not connect this with MCP. But too many parallel tasks started in quick succession might have contributed to it.

**Comment by luizaogs** [comment]
Hm ok, but this is still odd -- all other websites continue to work, but I have PVD trying to download and it's taking forever (because of the bad hotel internet) and while that is happening, I still can't access any other Kiosk. They don't load.

**Comment by luizaogs** [comment]
Otherwise I've tested stuff and it seems ok. Haven't gotten to synchronization yet, though, due to the slow download.

**Comment by urapadmin** [comment]
since MCP is not involved in the download I close this. If the freezing of Kiosk happens again in the future, we open its own ticket for that. Currently I attribute it to Hotel issues.
